docs/_includes/common/icon-button.html

```diff
@@ -54,4 +54,48 @@ <h3 id="icon-button-badged">Badged Icon Button</h3>
 </button>
     {% endhighlight %}
 
+    <h3 id="icon-button-cart-nav"> Cart Nav Icon Button </h3>
+    <p>DS Evo includes a new cart icon button with a grey background.</p>
+    <p>It requires an additional couple classes on the button - icon-btn--nav icon-btn--cart - as well as the icon-cart class on the svg.</p>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <button aria-label="menu" type="button" class="icon-btn icon-btn--badged icon-btn--nav icon-btn--cart">
+                <svg class="icon icon--cart" aria-hidden="true" focusable="false" width="24" height="24"><use xlink:href="#icon-cart"></use></svg>
```

Comment:
There seems to be a problem with the icons. Can you be sure to use the Jekyll include, e.g.:

```
{% include common/symbol.html name="settings" %}
```

![Screen Shot 2021-06-21 at 6 02 13 PM](https://user-images.githubusercontent.com/38065/122846522-dc7f1c80-d2ba-11eb-8987-3410ec2691f4.png)